FAQ.md

```diff
@@ -22,3 +22,24 @@ If multiple users are running on a system with a shared Tenstorrent device,
 you may encounter a `PermissionError: [Errno 13] Permission denied: '/tmp/*.lock'`.
 
 You would need to remove these files between user sessions.
+
+## Python pip install dependency resolution issues
+
+Your Python environment's pip version, if different than required version, have different [dependency resolution](https://pip.pypa.io/en/stable/topics/dependency-resolution/) and not correctly install pybuda (tt-buda) or its dependencies.
+To avoid dependency resolution issues check you are using the supported and tested version of pip for your tt-buda release. For example, pip==24.0 is required for the most recent release (v0.10.5). To avoid changing the system installation of pip we generally recommend using the venv installation method (see guide in https://github.com/tenstorrent/tt-buda-demos/blob/main/first_5_steps/1_install_tt_buda.md#python-environment-installation).
```

Comment:
```suggestion
To avoid dependency resolution issues, check that you are using the supported and tested version of `pip` for your `pybuda` release. For example, `pip==24.0` is required for the most recent releases. To avoid changing the system installation of `pip` we generally recommend using the venv installation method (see guide in https://github.com/tenstorrent/tt-buda-demos/blob/main/first_5_steps/1_install_tt_buda.md#python-environment-installation).
```

Reply:
Thanks for the copy review! Just added these changes.